main/src/main/java/de/blinkt/openvpn/VpnProfile.java

```diff
@@ -61,7 +61,7 @@
     public static final String INLINE_TAG = "[[INLINE]]";
     public static final String DISPLAYNAME_TAG = "[[NAME]]";
     public static final int MAXLOGLEVEL = 4;
-    public static final int CURRENT_PROFILE_VERSION = 7;
+    public static final int CURRENT_PROFILE_VERSION = 8;
```

Comment:
This is unecessary for new booleans that are false.

Reply:
I updated the code with a squashed commit